test/jdk/java/awt/print/PrinterJob/PrintNullString.java

```diff
@@ -126,7 +84,7 @@ private void paint(Graphics2D g2d) {
             // API 1: null & empty drawString(String, int, int);
```

Comment:
Now this `paint` method is redundant, you can copy its body directly into `print` above.